content/blog/2021/spending_policy_demo.md

```diff
@@ -0,0 +1,435 @@
+}
+```
+
+## Step 2a: Send testnet bitcoin from a faucet to receive address
+
+After a faucet payment is sent, use a testnet block explorer to confirm the transaction was included in a block. 
+
+[https://mempool.space/testnet/address/tb1qpqglt6yntay0se5vj3a7g36rql5pyzzp0w6jknfch2c0unwphsxs22g96e](https://mempool.space/testnet/address/tb1qpqglt6yntay0se5vj3a7g36rql5pyzzp0w6jknfch2c0unwphsxs22g96e)
+
+## Step 3a: Sync participant wallets and confirm balance
+
+This step must be done by Alice, Bob, and Carol so their individual descriptor wallets know about the faucet transaction they will later be spending the output of.
+
+```bash
+bdk-cli wallet -w alice -d $ALICE_DESCRIPTOR sync       
+{}
+bdk-cli wallet -w alice -d $ALICE_DESCRIPTOR get_balance
+{
+  "satoshi": 10000
+}
+
+bdk-cli wallet -w bob -d $BOB_DESCRIPTOR sync       
+{}
+bdk-cli wallet -w bob -d $BOB_DESCRIPTOR get_balance
+{
+  "satoshi": 10000
+}
+
+bdk-cli wallet -w carol -d $CAROL_DESCRIPTOR sync       
+{}
+bdk-cli wallet -w carol -d $CAROL_DESCRIPTOR get_balance
+{
+  "satoshi": 10000
+}
+```
+
+## Step 4a: View wallet spending policies
+
+This can also be done by any wallet participant, as long as they have the same descriptor and extended public keys from the other particpants..
+
+```bash
+bdk-cli wallet -w alice -d $ALICE_DESCRIPTOR policies
+{
+  "external": {
+    "contribution": {
+      "conditions": {
+        "0": [
+          {}
+        ],
+        "3": [
+          {
+            "csv": 2
+          }
+        ]
+      },
+      "items": [
+        0,
+        3
+      ],
+      "m": 3,
+      "n": 4,
+      "type": "PARTIAL"
+    },
+    "id": "tjdhrksc",
+    "items": [
+      {
+        "contribution": {
+          "condition": {},
+          "type": "COMPLETE"
+        },
+        "fingerprint": "5adb4683",
+        "id": "uyxvyzqt",
+        "satisfaction": {
+          "type": "NONE"
+        },
+        "type": "SIGNATURE"
+      },
+      {
+        "contribution": {
+          "type": "NONE"
+        },
+        "fingerprint": "4331a5e6",
+        "id": "w0xldflx",
+        "satisfaction": {
+          "type": "NONE"
+        },
+        "type": "SIGNATURE"
+      },
+      {
+        "contribution": {
+          "type": "NONE"
+        },
+        "fingerprint": "eebab672",
+        "id": "ghlsq5yk",
+        "satisfaction": {
+          "type": "NONE"
+        },
+        "type": "SIGNATURE"
+      },
+      {
+        "contribution": {
+          "condition": {
+            "csv": 2
+          },
+          "type": "COMPLETE"
+        },
+        "id": "8kel7sdw",
+        "satisfaction": {
+          "type": "NONE"
+        },
+        "type": "RELATIVETIMELOCK",
+        "value": 2
+      }
+    ],
+    "satisfaction": {
+      "type": "NONE"
+    },
+    "threshold": 3,
+    "type": "THRESH"
+  },
+  "internal": null
+}
+```
+
+## Step 5a: Create spending transaction
+
+The transaction can also be created by Alice, Bob, or Carol.
```

Comment:
Suggested additon:

The transaction can also be created by Alice, Bob, or Carol, or any third party that has all 3 `tpub`. Note that the argument provided to the `--external_policy` flag contains the id retrieved from the `policies` subcommand, in this case `tjdhrksc`.

Reply:
This is a good point, and it is common to have an untrusted coordinator who would only have pub keys.